src/software/ai/hl/stp/play/shoot_or_pass/shoot_or_pass_play_fsm.cpp

```diff
@@ -146,13 +146,49 @@ void ShootOrPassPlayFSM::takePass(const Update& event)
 
 bool ShootOrPassPlayFSM::passFound(const Update& event)
 {
-    return best_pass_and_score_so_far.rating > min_pass_score_threshold;
+    if (event.common.world.ball().velocity().length() < 0.5)
```

Comment:
woah this is a really good catch and a clever solution

From what I understand (from our discussion offline as well), we want to still look for passes when the ball is moving, but we don't want to take any of them until the ball speed is interceptable by a robot.

I've been testing this a bunch, and it works _great_ the gameplay is exciting and we always score

can't explain how excited I am to try this out at robocup

can you also make 0.5 a dynamic parameter